exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+var enoughSpaceInRoom = studentCount + mentorCount === roomMaxCapacity; // finish this statement
```

Comment:
Your expression for `enoughSpaceInRoom` is true when the studentCount is 16 and the mentorCount is 9, but doesn't work if the studentCount is changed to 10.
```
studentCount = 10;
mentorCount = 9;
roomMaxCapatcity = 25;

studentCount + mentorCount === roomMaxCapacity 
10 + 9 === 25 -> false
```
How can you change this expression so that it works if the student or mentor count number changes?

Reply:
You are right @louisechow , thanks for this suggestion, I should use `>=` in this case.